gatorgrade/generate/generate.py

```diff
@@ -14,8 +15,29 @@ def create_targeted_paths_list():
     # Add those targeted file paths into a list and returns it.
 
 
-def write_yaml_of_paths_list():  # expected input: A path list
+def write_yaml_of_paths_list(path_names):  # expected input: A path list
     """Write YAML file to create gatorgrade file and set default messages."""
-    # Create a new YAML file with PyYaml in the specific path.
-    # Write the default set up messages in YAML file.
-    # List the file paths in specific format.
+    files_list = []
+    # Create an empty list to store dictionaries
+    for file_path in path_names:
+        # Iterate through items in path_names
+        file_path_fixed = file_path.replace("./", "")
```

Comment:
I discussed this with @Yanqiao4396 and a few others during class today -- I think you'll need to make sure that you're removing the "root directory" value that is given to `generate_config` here, instead of just `.`, which is only the default.

Reply:
This step might actually work better if it's extracted to its own (small) function, which is then called in `generate_config` before the list of paths even get here.

Reply:
Thanks for your input!